src/controllers/scriptController.js

```diff
@@ -28,9 +28,24 @@ const scriptController = {
     } catch (error) {
       return next({
         log: `error: ${error} occurred when getting actor's characters from the db.`,
-        message: 'error in getActorCharacters in actorController.',
+        message: 'error in getActorCharacters in actorController.'
+      });
+    }
+  },
+
+  assignCharacter: async (req, res, next) => {
+    const { scriptId } = req.params;
+    const { actorId, characterId } = req.body
+    try {
+      res.locals.assignedCharacter = await scriptRepo.assignCharacter(scriptId, characterId, actorId);
```

Comment:
Do we need a characterRepo? Seems like this doesn't really fit in either script or actor

Reply:
Added a characterRepo and characterController. Rerouted the getCharacters and assignCharacter endpoints to said controller and repo.